### ISSUE TITLE
JAWS 2019 doesn't read graphics in Firefox

### ISSUE THREAD

**Issue by davidengebretson** [FreedomScientific/standards-support#137]
### Summary
JAWS 2019 doesn't see graphics on page in Firefox.
### Expected result
I expected JAWS to announce the first graphic on the page.
### Actual result
JAWS says "no graphics"

### Example
1. go to https://wwu.edu in Firefox 63.0.3 with JAWS 2019
2. Press 'g' to jump to the first graphic

### Additional Information
Graphics are read using JAWS 2018
#### JAWS version and build number
All versions of JAWS 2019 

 
#### Operating System and version
Windows 10 1803

#### Browser and version:
Firefox 63.0.3

**Comment by stevefaulkner** [comment]
Hi @davidengebretson I have taken an initial look there is indeed something odd going on. 
As an aside there is an image with `alt=""` on the page and as it is the content of a link the filename is being announced:
`<img src="https://www.wwu.edu/sites/default/files/styles/slideshow/public/Chelsea%20and%20starfish2.jpg?itok=Utl2Koda&amp;timestamp=1542411627" alt="" title="" draggable="false">`

Furthermore the `alt` content for the images that are the sole content of links do not have appropriate text. Refer to [A link or button containing nothing but an image](https://www.w3.org/TR/html52/semantics-embedded-content.html#a-link-or-button-containing-nothing-but-an-image) guidance.

**Comment by stevefaulkner** [comment]
tested with JAWS 2023.2307.37 could not reproduce.
